Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM udacity/ruby:2.2.4
-RUN apk-install make gcc musl-dev
-RUN bundle install --jobs 20 --retry 5 --without development test
-ADD . /app
-ENV RACK_ENV production
+FROM ruby:2.6-alpine
```

Comment:
The Udacity Ruby Dockerfile isn't adding any value here, except perhaps that it's a pinned version.